.meta/sinks/http.toml.erb

```diff
@@ -103,6 +103,15 @@ type = "string"
 examples = ["https://10.22.212.22:9000/_health"]
 description = "A URI that Vector can request in order to determine the service health."
 
+[sinks.http.options.metrics_plus_mode]
```

Comment:
I'd prefer this to be removed. We already have an `encoding` option which supports json.